bioblend/_tests/TestGalaxyFolders.py

```diff
@@ -0,0 +1,58 @@
+"""
+Use ``nose`` to run these unit tests.
+"""
+from . import GalaxyTestBase, test_util
+
+FOO_DATA = 'foo\nbar\n'
+
+
+@test_util.skip_unless_galaxy("release_16.01")
```

Comment:
Can we be more granular with the test skipping? I think that you can remove this line and add the following method decorators:
- for `test_delete_folder()`: release_15.01
- for `test_update_folder()`: release_15.10
- for `test_get_set_permissions()`: release_16.01